src/main/java/com/hashedin/redmask/common/DataMasking.java

```diff
@@ -116,8 +117,26 @@ public void buildQueryAndAppend(MaskingRule rule, FileWriter writer,
             "Exception while appending masking function definition to file writer", e);
       }
     }
+    log.info("Masking data");
```

Comment:
improve log msg

Reply:
removed the message and split it inside the underlying method.